pandas/tests/computation/test_compat.py

```diff
@@ -11,8 +11,6 @@
 from pandas.computation.engines import _engines
 import pandas.computation.expr as expr
 
-ENGINES_PARSERS = list(product(_engines, expr._parsers))
```

Comment:
There's only one parametrized function in this file, so maybe it's not so useful here.